### DISCUSSION TITLE
Scratch Scroll Fix Addon

### DISCUSSION THREAD

**Discussion by MiningMiner27** [ScratchAddons/ScratchAddons#5741] [Ideas]
An addon that would fix Scratch's _occasionally_ wonky scrolling in the editor would be cool. The scrolling on Scratch sometimes breaks, causing the scrolling to either be amplified, dampened, or scroll in an unexpected way.

Example: You want to scroll up, so you scroll up on your scroll wheel, but it scrolls slightly to the left. So you then scroll right by holding shift and scrolling down, but the scrolling is slower than expected. You then scroll down by scrolling up on your scroll wheel, but it scrolls faster than expected. you then scroll left by holding shift and scrolling up, but it scrolls slightly downwards.

So I'd like an addon to help fix that. I'd appreciate it.

**Comment by DNin01** [comment]
I think this happens only when there are comments in code. Scratch's code comments do introduce some bugs.

I wonder if we could update the Better editor comments addon to offer a setting to fix this?
